src/mnelab/model.py

```diff
@@ -43,8 +43,8 @@ def wrapper(*args, **kwargs):
 class Model:
     """Data model for MNELAB."""
 
-    def __init__(self):
-        self.view = None  # current view
+    def __init__(self, view=None):
+        self.view = view  # current view, default = None
```

Comment:
I actually just noticed that this does not work for initializing our model in one step, because the view requires a model, and so the model cannot require a view. So maybe let's leave it as is, meaning that by default `model.view = None`, there is no specific parameter needed. Sorry about the confusion!